test/unit/UnitTests.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 	<head>
 		<meta charset="utf-8">
 		<title>ThreeJS Unit Tests - Using Files in /src</title>
 		<link rel="stylesheet" href="../../node_modules/qunit/qunit/qunit.css">
+		<!-- if we don't set favicon here we have to intercept the request for it in puppeteer page to prevent console error -->
+		<link rel="icon" type="image/x-icon" href="/files/favicon.ico" />
```

Comment:
Every time I run the tests I saw a console error : `The resource could not be loaded , server returned 404`. It was due to this favicon. Initially I intercepted it in the new `puppeteer.unit.js` and returned /files/favicon.ico for that specific request, then I realized it is less trouble to add it here.